### PR TITLE
module able to accept custom AcceptEnv to sshd - Fixing issue #288

### PR DESCRIPTION
**Pull request by renato-farias**

### CODE REVIEW

**Comment by renato-farias** [comment]
@ghoneycutt I fixed the mess. Please take a look.

**Comment by dhose** [comment]
Any plans on merging this PR? 🙂

File: manifests/init.pp [hunk 1]
```diff
@@ -446,6 +447,15 @@
     }
   }
 
+  case type3x($sshd_acceptenv_list) {
```
Comment (ghoneycutt) [inline manifests/init.pp]: you could just use `validate_array()`

File: spec/fixtures/sshd_config_debian [hunk 2]
```diff
@@ -97,9 +97,20 @@ GSSAPICleanupCredentials yes
 UsePAM yes
 
 # Accept locale-related environment variables
-AcceptEnv LANG LC_CTYPE LC_NUMERIC LC_TIME LC_COLLATE LC_MONETARY LC_MESSAGES
```
Comment (ghoneycutt) [inline spec/fixtures/sshd_config_debian]: You can't change the fixtures files, they are meant to be static

File: manifests/init.pp (hunk 1 above)
Comment (renato-farias) [reply manifests/init.pp]: validate_array Deprecated: Will be removed in a future version of stdlib. See validate_legacy.

File: spec/fixtures/sshd_config_debian (hunk 2 above)
Comment (renato-farias) [reply spec/fixtures/sshd_config_debian]: I can't see any problem on it. It is according the module will generate the files.

**Pull request closed without merging**